packages/suite/src/actions/wallet/send/sendFormBitcoinActions.ts

```diff
@@ -215,7 +220,11 @@ export const signTransaction =
             signEnhancement.outputs = restoreOrigOutputsOrder(transaction.outputs, outputs, txid);
         }
 
-        const signPayload = {
+        if (!device.unavailableCapabilities?.amountUnit) {
```

Comment:
on second thought, lets keep the condintion, but change it to:
```
if (network.feature['amount-unit'] && !device.unavailableCapabilities?.amountUnit) {
```
because of invalid case:
i do have enabled bitcoin and litecoin and i've changed amount units for bitcoin (but not for litecoin)